README.md

```diff
@@ -466,6 +466,35 @@ To build an Amazon-like autocomplete menu, suggesting popular queries and for th
 }
 ```
 
+##### User-generated data: protecting against XSS
+
+Malicious users may attempt to engineer XSS attacks by storing HTML/JS in their data. It is important that user-generated data be properly escaped before using it in an autocomplete.js template.
+
+The `enableXSSProtection` option can be set to `true` to provide an `escapeHTML` function as third argument to the suggestion template function. It can be used in the following way:
+
+```js
+$('#states').autocomplete({}, [
+  {
+    displayKey: 'name',
+    source: $.fn.autocomplete.sources.hits(index, {
+      hitsPerPage: 5,
+      enableXSSProtection: true,
+      highlightPreTag: "<strong style=\"red\">",
+      highlightPostTag: "</strong>"
+    }),
+    templates: {
+      suggestion: function(suggestion, params, escapeHTML) {
+        var val = suggestion._highlightResult.name.value;
+        return escapeHTML(val);
```

Comment:
What we did in react-instantsearch was to provide a method like that:

```
const highlightedTitle = highlight({attributeName: 'desc.title', hit})
return `Hello ${highlightedTitle}`;
```

So that it provides:
- an XSS protection
- an easy way to use highlighting without the struggle of digging into `_highlightResult.name.value`.

Do you think it would be valuable here? If so I would provide both highlight() and escapeHTML(). Document both usages and intents. escapeHTML would be our small util that we bundle inside our libs so the users willing to go full secure (our needing it) can do it.

(As for desc.title, we use lodash.get to easily go through the _highlightResult nested object).